$NetBSD: patch-panel_CMakeLists.txt,v 1.4 2024/08/02 17:54:16 pin Exp $

Work around undefined symbol issue (which would cause the panel to fail
to run if the volume plugin was enabled), as discussed in:
https://github.com/lxqt/lxqt/issues/1075

Install fallback .conf files to a consistent location.

--- panel/CMakeLists.txt.orig	2024-06-07 07:16:13.376894114 +0000
+++ panel/CMakeLists.txt
@@ -58,6 +58,8 @@ set(LIBRARIES
     lxqt
 )
 
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Xlinker --export-dynamic")
+
 file(GLOB CONFIG_FILES resources/*.conf)
 
 ############################################
@@ -110,7 +112,7 @@ target_link_libraries(${PROJECT}
 set_property(TARGET ${PROJECT} PROPERTY ENABLE_EXPORTS TRUE)
 
 install(TARGETS ${PROJECT} RUNTIME DESTINATION bin)
-install(FILES ${CONFIG_FILES} DESTINATION ${CMAKE_INSTALL_DATADIR}/lxqt)
+install(FILES ${CONFIG_FILES} DESTINATION share/examples/lxqt)
 install(FILES ${PUB_HEADERS} DESTINATION include/lxqt)
 install(FILES
     man/lxqt-panel.1
